source/kraken/configuration.cpp

```diff
@@ -80,6 +80,7 @@ po::options_description get_options_description(const boost::optional<std::strin
 
         ("GENERAL.enable_request_deadline", po::value<bool>()->default_value(true), "enable deadline of request")
         ("GENERAL.metrics_binding", po::value<std::string>(), "IP:PORT to serving metrics in http")
+        ("GENERAL.core_file_size_limit", po::value<int>()->default_value(0), "ulimit that define the maximum size of a core file")
```

Comment:
can it be size_t by any chance?

Reply:
nope, as you can have `-1` to set the value to `unlimited`

Reply:
So `0` is the system's limit right ?
Can you either comment it or precise in description ?

Maybe also, precise the size is in blocks (from `ulimit -c`) or bytes (from trace below) ?

Reply:
nope, `0` is absolutely valid. it tells the system that no core dump should be performed.